utils/webrtc/webrtc.js

```diff
@@ -24,6 +24,21 @@ function generateUID() {
 }
 
 
+export function isScreenShareSourceAvailable() {
+  // currently we only support chrome v70+ (w/ experimental features enabled, if necessary)
+  // and firefox
+  return (navigator.getDisplayMedia ||
+    navigator.mediaDevices.getDisplayMedia ||
+    !!navigator.mediaDevices.getSupportedConstraints().mediaSource);
+}
+
+export const logger = {
```

Comment:
This does not seem like a reasonable place for this utility function. Logging isn't specific to just the webrtc code.
We do a lot of logging in the riff specific code we've been adding to mattermost. Currently it is mostly done using `console.log`. That should change in the future to use the `logger`.
I think the logger should live either in `utils/riff/index.js` or in `utils/riff/logger.js` and be imported as either
```
import { logger } from '../utils/riff';
```
or
```
import { logger } from '../utils/riff/logger';
```

`utils/riff` is reasonable since it looks like we are making some attempt to keep the riff code separate from the base mattermost code.

Reply:
Ok that makes sense. I've made that change.